lib/__tests__/index-test.js

```diff
@@ -353,7 +353,7 @@ describe('labels', () => {
 
   it('warns if an image without alt is the only content', () => {
     expectWarning(assertions.render.NO_LABEL.msg, () => {
-      <button><img src="#" alt=""/></button>;
+      <button><img src="#"/></button>;
```

Comment:
I just repaired the test already there, for which it says "without alt" while still providing a alt with value empty string. The point of this PR is to fix an erroneous and inconsistent `alt` attribute detection, an empty alt and no alt are not the same thing.